packages/communication-react/review/communication-react.api.md

```diff
@@ -35,6 +35,8 @@ import { IContextualMenuItem } from '@fluentui/react';
 import { IStyle } from '@fluentui/react';
 import { MediaStreamType } from '@azure/communication-calling';
 import { MicrosoftTeamsUserKind } from '@azure/communication-common';
+import { OutputParametricSelector } from 'reselect';
```

Comment:
nit: why not the following?

import { OutputParametricSelector, OutputSelector } from 'reselect';

Reply:
I think due to a small bug in the fix for the api-extractor original bug, but not something we have control over